qiskit_aqua/algorithms/adaptive/qsvm/qsvm_variational.py

```diff
@@ -226,6 +236,31 @@ def _get_prediction(self, data, theta):
 
         return predicted_probs, predicted_labels
 
+    # Breaks data into minibatches. Labels are optional, but will be broken into batches if included.
+    def batch_data(self, data, labels=None):
```

Comment:
it will be better you add `minibatch_size` as an argument, then it will be easier to configure batches with different size in train, test and predict.